parser/openfoodfacts_taxonomy_parser/parser/taxonomy_parser.py

```diff
@@ -503,14 +509,24 @@ def _merge_duplicate_entry_nodes(self, entry_nodes: list[NodeData]) -> list[Node
         return unique_entry_nodes
 
     def _create_taxonomy(
-        self, filename: str, external_entry_nodes: list[NodeData] | None = None
+        self, filename: str, external_filenames: list[str] | None = None
     ) -> Taxonomy:
         """Create the taxonomy from the file"""
+        # parse external taxonomies if any, and add their entry nodes to the main taxonomy
+        external_entry_nodes = []
+        for external_filename in external_filenames or []:
+            external_taxonomy_parser = TaxonomyParser()
+            external_taxonomy = external_taxonomy_parser.parse_file(
+                external_filename, None, self.parser_logger
+            )
```

Comment:
I believe this should also work and may be cleaner
```suggestion
            external_taxonomy = self.parse_file(
                external_filename, None, self.parser_logger
            )
```

Reply:
I don't think so, because of the `self.stopwords` dict that is constructed during `_harvest_entries`